src/coreclr/src/jit/codegencommon.cpp

```diff
@@ -4756,9 +4756,16 @@ void CodeGen::genCheckUseBlockInit()
     CLANG_FORMAT_COMMENT_ANCHOR;
 
 #ifdef TARGET_64BIT
+#if defined(FEATURE_SIMD) && ALIGN_SIMD_TYPES
```

Comment:
This will also change the heuristic for arm64.

```suggestion
#if defined(`TARGET_AMD64`)
```

Reply:
Changed